robot-tests/check_results.py

```
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+"""
+Quick script to check Robot Framework test results
+"""
+
+import os
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)